Rpackage/R/mmit.cv.R

```diff
@@ -0,0 +1,104 @@
+#' The Cross Validation of Max Margin Interval Tree
```

Comment:
Not a big deal, but I'm curious: why is the loss type 0 or 1 in the cv_results instead of "hinge" or "squared"?

Reply:
it happens after the foreach statement, so I guess thats the reason, fit_and_score gives strings only, i dont know why it changed to labels here. maybe i'll have to manually change it back